### ISSUE TITLE
Che addon uses an outdated template

### ISSUE THREAD

**Issue by l0rd** [minishift/minishift#3010]
### General information

  * Minishift version: v1.27.0+707887e
  * OS: macOS
  * Hypervisor: xhyve

### Steps to reproduce

  1. Start Che (`minishift addon apply che`)
  2. Create and Open a workspace using `Che 7` stack

### Expected

 A Che 7 workspaces is created

### Actual

The workspace fails to start

**Comment by l0rd** [comment]
@agajdosi how can I run Che addon integration tests for this PR?

**Comment by agajdosi** [comment]
@l0rd in the Makefile, the is target `make integration_pr`, to run che tests, please change this line https://github.com/minishift/minishift/blob/master/Makefile#L281 to:

```
integration_pr: GODOG_OPTS = --tags=addon-che,addon-che-with-anyuid
```

this will just ran the two che tests. We still do not have automation through comments on GH of this. So once it passes, you can just comment link to the build and remove the line.

**Comment by l0rd** [comment]
@agajdosi I have done that and it worked fine on centos.ci (I had to fix a test first): https://ci.centos.org/job/minishift-pr/5415/

I'm also interested in running those tests locally to extend them with a Che 7 stack but I am not able to run them successfully. I have tried to run:

```bash
make integration_pr
```

but it fails with the following error

```
  Scenario: User starts Minishift                          # features/addons/addon-che-with-anyuid.feature:14
    Given Minishift has state "Does Not Exist"             # testsuite.go:77 -> *Minishift./testsuite.shouldHaveState
    When executing "minishift start --memory=5GB" succeeds # testsuite.go:649 -> /testsuite.ExecutingMinishiftCommandSucceedsOrFails
    Command 'start --memory=5GB' did not execute successfully. cmdExit: 1, cmdErr: Error pulling the OpenShift container image: ssh command error:
command : docker pull openshift/origin-control-plane:v3.11.0
err     : exit status 1
output  : Trying to pull repository docker.io/openshift/origin-control-plane ...
Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 192.168.64.1:53: read udp 192.168.64.33:53754->192.168.64.1:53: read: connection refused
```

Any idea on the cause of the error. I am running the tests on a macOS if that makes any difference.

**Comment by agajdosi** [comment]
@l0rd Test are prepared to be executed on Linux, Mac and Windows, so it should not be a problem itself to run it on Mac. The error comes from minishift command, so I am wondering whether the issue is persistent on the machine, also are you able to start the minishift normally from the terminal on that machine? It might be temporary network problem or firewall blocking the connection.

**Comment by l0rd** [comment]
@agajdosi yes I am able to start minishift locally (`./cmd/minishift`) and I can `minishift ssh` and pull `openshift/origin-control-plane:v3.11.0`. I have tested on fedora and it hangs at step "Given Minishift has state "Does Not Exist"\".

**Comment by agajdosi** [comment]
@l0rd So on Mac you can start locally but from the integration tests the minishift start can't start permanently. Once it fails on the start, you can stop the tests, set MINISHIFT_HOME_DIRECTORY to the `out/test-run/.minishift` of the Minishift repository and do minishift ssh to the test VM to see whether it can do the pull.

About the Fedora: when the tests shows green `Given Minishift has state "Does Not Exist"` it means that step was successful and actually it is executing the next step - I guess it could be `minishift start`. Did you give it enough time to start?

**Comment by l0rd** [comment]
> So on Mac you can start locally but from the integration tests the minishift start can't start permanently. Once it fails on the start, you can stop the tests, set MINISHIFT_HOME_DIRECTORY to the out/test-run/.minishift of the Minishift repository and do minishift ssh to the test VM to see whether it can do the pull.

Ok cool will try that

> About the Fedora: when the tests shows green Given Minishift has state "Does Not Exist" it means that step was successful and actually it is executing the next step - I guess it could be minishift start. Did you give it enough time to start?

Yes I have waited one hour or so...
